src/Geom_Graph.f90

```diff
@@ -294,32 +361,32 @@ ELEMENTAL FUNCTION getAdjacentVert_graphType(thisGraph,v0,i) RESULT(v1)
 !>
 !>
 !>
-ELEMENTAL FUNCTION getCWMostVert_graphType(thisGraph,v0,vCurr) RESULT(vNext)
-  CLASS(GraphType),INTENT(IN) :: thisGraph
-  INTEGER(SIK),INTENT(IN) :: vCurr
+ELEMENTAL FUNCTION getCWMostVert_graphType(this,v0,vCurr) RESULT(vNext)
```

Comment:
Looks like you got a lot of Doxygen added but some is still missing.

Reply:
Yeah... there are a handful (including this one) that I simply don't know exactly what they're doing because I didn't really have to rely on them at all.  I just left them blank since I don't want to put something wrong in there.

Reply:
Understood...I think we should at least try to tag them with an issue or something.

Reply:
I can get on board with that

Reply:
#303 has it